reduxApp/dev/js/components/README.md

```diff
@@ -0,0 +1,5 @@
+## Containers vs. Components
```

Comment:
@VARTIKASINGH1234 why you have added the read-me files here. Please delete that files when you're doing copy-paste some code.